### PR TITLE
Add recommendations section at the end

### PR DESCRIPTION
**Pull request by qwrtln**
en-37
en-38
en-39

### CODE REVIEW

File: sections/recommendations.tex [hunk 1]
```diff
@@ -0,0 +1,110 @@
+% !TeX spellcheck = en_US
+\addsection{Recommendations}{\art/haste.png}
+
+\iftoggle{printable}{}{\bigbreak}
+
+\begin{multicols}{2}
+We've gathered up some custom rules and best practices to make your gaming experience as smooth as possible.
```
Comment (tomaas-zeman) [inline sections/recommendations.tex:7]:
```suggestion
We've gathered some custom rules and best practices to make your gaming experience as smooth as possible.
```

File: sections/recommendations.tex [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+Bear in mind that those are just \textbf{opinions of the authors} of this book.
+Enjoy.
+
+\subsection*{Play Time}
+
+You can estimate your \textbf{minimal} play time using the following formula:
```
Comment (tomaas-zeman) [inline sections/recommendations.tex:13]: I like the idea of highlighting the minimum 👍

**Review by tomaas-zeman — APPROVED** [review]

File: sections/recommendations.tex [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+Where:
+\begin{itemize}
+  \item[$\textbf{P}$] -- number of players
+  \item[$\textbf{R}$] -- number of rounds
+\end{itemize}
+
+Therefore, if you're playing a 3-player scenario consisting of 12 rounds, you can easily calculate that $3 × 12 × 5 = 180$.
+Your expected minimal play time should be around 3 hours.
+To calculate the \textbf{upper bound} of your play time (more realistic if there are inexperienced players), just double the result.
```
Comment (krazanas) [inline sections/recommendations.tex:27]:
```suggestion
To calculate the \textbf{upper bound} of your play time (more realistic if there are inexperienced players), simply double the result.
```

File: sections/recommendations.tex [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+\subsection*{Trading Post}
+
+Although rules as written state that you can remove only one card from your hand while visiting a Trading Post, it is recommended to allow removing as many as you like.
+
+\includegraphics[width=\linewidth]{\art/rogue.jpg}
+
+\subsection*{Combat with Neutral Units}
+
+During combat with Neutral Units in clash sccenarios, if a player decides to deploy their Units only on one half (left or right one) of the battle board, the player positioning the Neutral Units must follow suit.
```
Comment (krazanas) [inline sections/recommendations.tex:43]:
```suggestion
During combat with Neutral Units in clash scenarios, if a player decides to deploy their Units only on one half (left or right one) of the battle board, the player positioning the Neutral Units must follow suit.
```

File: sections/recommendations.tex [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+  \begin{minipage}[t]{0.44\textwidth}
+    \centering
+    \includegraphics[width=\linewidth]{\examples/battle_good.png}
+    \caption[good protected]{\textit{Neutral Units are positioned correctly.}}
+  \end{minipage}
+  \hfill
+  \begin{minipage}[t]{0.44\textwidth}
+    \centering
+    \includegraphics[width=\linewidth]{\examples/battle_bad.png}
+    \caption[bad protected]{\textit{This is not allowed, as the Necropolis player deployd their Units on the left half of the Combat Board.
```
Comment (krazanas) [inline sections/recommendations.tex:67]:
```suggestion
    \caption[bad protected]{\textit{This is not allowed, as the Necropolis player deployed their Units on the left half of the Combat Board.
```

File: sections/recommendations.tex (hunk 2 above)
Comment (qwrtln) [reply sections/recommendations.tex:13]: Still thinking about that table 🤔

**Comment by qwrtln** [comment]
po4a commit

**Pull request merged**